test/specs/commonTests.js

```diff
@@ -385,6 +388,9 @@ export const implementsAlignedProp = (Component, requiredProps = {}) => {
         it('adds "justified" without "aligned" to className', () => {
           shallow(<Component { ...requiredProps } aligned='justified' />)
             .should.have.className('justified')
+
+          shallow(<Component { ...requiredProps } aligned='justified' />)
+            .should.not.have.className('aligned')
```

Comment:
Nice catch.  Let's reuse use the chai `and` chain instead of re-rendering though:

```js
shallow(<Component { ...requiredProps } aligned='justified' />)
  .should.have.className('justified')
  .and.should.have.className('aligned')
```